_maps/map_files/BirdStation/job_changes.dm

```diff
@@ -0,0 +1,159 @@
+#define JOB_MODIFICATION_MAP_NAME "BirdStation"
+
+//Removed jobs
+/datum/job/atmos/map_check()
+	return FALSE
+
+/datum/job/chief_engineer/map_check()
+	return FALSE
+
+/datum/job/qm/map_check()
+	return FALSE
+
+/datum/job/cmo/map_check()
+	return FALSE
+
+/datum/job/chemist/map_check()
+	return FALSE
+
+/datum/job/geneticist/map_check()
+	return FALSE
+
+/datum/job/virologist/map_check()
+	return FALSE
+
+/datum/job/rd/map_check()
+	return FALSE
+
+/datum/job/roboticist/map_check()
+	return FALSE
+
+/datum/job/chaplain/map_check()
+	return FALSE
+
+/datum/job/warden/map_check()
+	return FALSE
+
+/datum/job/lawyer/map_check()
+	return FALSE
+
+//Job changes
+
+/datum/job/New()
+	..()
+	supervisors = "the captain and the head of personnel"
+
+
+/datum/job/assistant // Here so assistant appears on the top of the select job list.
+
+//Access Changes + Flavo(u)r
+//Command
+
+/datum/job/captain/New()
+	..()
+	supervisors = "Nanotrasen and Central Command"
+
+/datum/job/hop/New()
+	..()
+	supervisors = "the captain and Central Command"
+
+/datum/job/hop/get_access()
+	return get_all_accesses()
```

Comment:
MAP_JOB_CHECK

Reply:
This needs to be corrected however
```dm
. = ..()
MAP_JOB_CHECK
return get_all_accesses()
```

Reply:
Are you sure it's not supposed to be MAP_JOB_CHECK_BASE like how Omega did it though?